test/e2e/src/support/hooks.ts

```diff
@@ -33,9 +33,14 @@ After(async function () {
   const world = this as World
   world.client.disconnect()
 
-  // This will be removed once unloading is implemented
-  const docker = new Docker({ socketPath: '/var/run/docker.sock' })
-  const containers = await docker.listContainers()
-  const targetContainers = containers.filter((container) => container.Image.match(/samjeston/g) || container.Image.match(/jann/g))
-  await Promise.all(targetContainers.map(c => docker.getContainer(c.Id).kill()))
+  const contracts = await world.client.contracts()
+  await Promise.all(contracts.map((c: any) => world.client.unloadContract(c.contractAddress)))
+
+  if (process.env.TEST_MODE === 'docker') {
+    // This will be removed once unloading is implemented
```

Comment:
Unloading is implemented in this PR right?

Reply:
Indeed it is!